### ISSUE TITLE
Unit tests: Google Chrome

### ISSUE THREAD

**Issue by dbrekelmans** [dbrekelmans/browser-driver-installer#8]
We need unit tests for:
- `DBrekelmans\BrowserDriverInstaller\Browser\GoogleChrome\PathResolver`
- `DBrekelmans\BrowserDriverInstaller\Browser\GoogleChrome\VersionResolver`

**Comment by laurent35240** [comment]
Hi,

I opened Pull request #30 for adding Google Chrome Unit tests.

Two stuffs I found while writing them:

1. I could not mock class `OperatingSystem` because it is a final class. Because of that I could not test the case when we provide an unknown Operating System. For testing it properly this part we must either add interface or remove final from this class
2. Having `phpcs`, `psalm` and `phpstan` can make it harder to write code because these 3 tools can sometime disagree with each other. I needed to add `ignoreErrors` in `phpstan` for not having errors about using Symfony Process
